app/forms/mappings/LocalDateFormatter.scala

```diff
@@ -37,13 +38,15 @@ private[mappings] class LocalDateFormatter(
                                           ) extends Formatter[LocalDate] with Formatters {
 
   private val fieldKeys: List[String] = List("day", "month", "year")
-
+  private val theCurrentDate = LocalDate.now.format(DateTimeFormatter.ofPattern("d M yyyy"))
```

Comment:
This could go within the `toDate` method, as it is not used by any other method within the class.

If you want to leave as is, do not mind.

Reply:
updated.  Good spot